### PR TITLE
Update scripts to build CORD-19 index locally

### PR DESCRIPTION
**Pull request by edwinzhng**
We now use the new CORD-19 indexing script from Anserini to build, verify, and swap new indices. I also updated docs and pulled out the HNSW index update into a separate script.

### CODE REVIEW

File: scripts/update-anserini.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# Check if date provided
+if [ $# -eq 0 ]; then
+    echo "USAGE: sh scripts/update-anserini.sh [DATE]"
+    exit 1
+fi
+
+
+echo "Updating CORD-19 index for $1..."
+[ -d anserini ] && echo "Found Anserini folder..." || git clone https://github.com/castorini/anserini.git
+python3 -m pip install tqdm
```
Comment (rodrigonogueira4) [inline scripts/update-anserini.sh:22]: `tqdm` should be in the requirements already, right?

File: scripts/update-anserini.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```
Comment (rodrigonogueira4) [inline scripts/update-anserini.sh:1]: Should we rename this to `update-anserini-index.sh`? Update anserini sounds like updating Anserini's code.

File: README.md [hunk 3]
```diff
@@ -27,9 +25,14 @@ Install Java 11
 sudo apt-get install openjdk-11-jre openjdk-11-jdk
 ```
 
-Download the [latest Anserini indices](https://github.com/castorini/anserini/blob/master/docs/experiments-covid.md)
+Build the [latest Anserini indices](https://github.com/castorini/anserini/blob/master/docs/experiments-cord19.md)
+```
+sh scripts/update-anserini.sh
+```
+
+Build the latest HNSW index
```
Comment (rodrigonogueira4) [inline README.md:33]: nit: maybe add a short description of what is an hnsw?

File: scripts/update-anserini.sh (hunk 1 above)
Comment (edwinzhng) [reply scripts/update-anserini.sh:22]: It's not, but I'll add it there

**Pull request merged**